README.md

```diff
@@ -1,15 +1,28 @@
 # Resussun
 
-How to start the Resussun application
----
+## How to start the Resussun application
 
+### Use a local build
 1. Run `mvn clean install` to build your application
 1. Run Elasticsearch: `docker run -p 9200:9200 -p 9300:9300 -e "discovery.type=single-node" docker.elastic.co/elasticsearch/elasticsearch:7.5.2`
-1. Create an index: `curl -X POST --data "timbuctooUrl={timbuctooUrl}&dataSetId={dataSetId}" http://{resussun_host}:{admin_port}/tasks/createIndex`
 1. Start application with `./target/appassembler/bin/resussun server config.yml`
 1. To check that your application is running enter url `http://localhost:8080`
 
-Health Check
+### Use docker-compose.yml
+1. run the docker-compose file: `docker-compose -f docker/docker-compose.yml up -d`  
+1. To check that your application is running enter url `http://localhost:8080`
+
+### Use docker-compose-local.yml
+This docker-compose uses the local build to run with Elasticsearch.
+1. Run `mvn clean install` to build your application
+
+
+### Create an index
+1. Create an index: `curl -X POST --data "timbuctooUrl={timbuctooUrl}&dataSetId={dataSetId}" http://{resussun_host}:{admin_port}/tasks/createIndex`
```

Comment:
Dit vond ik in eerste instantie verwarrend. Ik moest even op zoek in de documentatie van DropWizard wat de admin port precies was of hoe ik dat kon aanpassen, voordat ik door had dat het by default port 8081 is.
Aangezien je overal verwijst naar 8080 of 8081 zou ik dat hier ook gewoon doen.

Reply:
Same for localhost vs resussun_host trouwens.